nixops/backends/hetzner.py

```diff
@@ -464,9 +464,12 @@ def _get_default_gw(self):
         """
         Return the default gateway of the currently running machine.
         """
-        cmd = "ip route list | sed -n -e 's/^default  *via  *//p'"
-        cmd += " | cut -d' ' -f1"
-        return self.run_command(cmd, capture_stdout=True).strip()
+        default_gw_cmd = "ip route list | sed -n -e 's/^default  *via  *//p'"
+        gw_ip_cmd = default_gw_cmd + " | cut -d' ' -f1"
+        gw_dev_cmd = default_gw_cmd + " | cut -d' ' -f3"
+        gw_ip = self.run_command(gw_ip_cmd, capture_stdout=True).strip()
+        gw_dev = self.run_command(gw_dev_cmd, capture_stdout=True).strip()
```

Comment:
Probably would make sense to use only one single command using `spli`t instead of the `cut`.

Reply:
Done